include/boost/geometry/algorithms/detail/overlay/get_turn_info.hpp

```diff
@@ -965,8 +922,57 @@ template
 >
 struct collinear : public base_turn_handler
 {
+    template
+    <
+        typename IntersectionInfo,
+        typename UniqueSubRange1,
+        typename UniqueSubRange2,
+        typename DirInfo
+    >
+    static bool handle_as_equal(IntersectionInfo const& info,
+                                UniqueSubRange1 const& range_p,
+                                UniqueSubRange2 const& range_q,
+                                DirInfo const& dir_info)
+    {
+#if defined(BOOST_GEOMETRY_USE_RESCALING)
+        return false;
+#endif
+        int const arrival_p = dir_info.arrival[0];
+        int const arrival_q = dir_info.arrival[1];
+        if (arrival_p * arrival_q != -1 || info.count != 2)
+        {
+            // Code below assumes that either p or q arrives in the other segment
+            return false;
+        }
+
+        auto const location = distance_measure(info.intersections[1],
```

Comment:
regarding the name `location` maybe `distance_from_location` or something is more relevant since this is not a location but some kind of distance.

Reply:
Agreed, I don't know why I took `location`. I will change it, and merge. Thanks for your review.

Reply:
In other places I used `dm` for this purpose so I use that now here too.